bin/modbus

```diff
@@ -5,15 +5,47 @@ import sys
 import os
 import logging
 
+import colorama as clr
+
 from modbus_cli.definitions import Definitions
 from modbus_cli.modbus_rtu import ModbusRtu
 from modbus_cli.modbus_tcp import ModbusTcp
 from modbus_cli.access import parse_accesses
 
 
+class ColourHandler(logging.Handler):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.formatter = logging.Formatter('%(style)s%(message)s'+clr.Style.RESET_ALL)
+
+    def emit(self, record):
+
+        if record.levelname == "DEBUG":
+            record.style = clr.Style.DIM
+        elif record.levelname == "WARNING":
+            record.style = clr.Style.BRIGHT
+        elif record.levelname == "ERROR":
+            record.style = clr.Style.BRIGHT+clr.Fore.RED
+        elif record.levelname == "CRITICAL":
+            record.style = clr.Style.BRIGHT+clr.Back.BLUE+clr.Fore.RED
+        else:
+            record.style = clr.Style.NORMAL
+
+        msg = self.format(record)
+
+        print(msg)
+
+
 def connect_to_device(args):
     if args.device[0] == '/':
-        modbus = ModbusRtu(args.device, args.baud, args.parity, args.stop_bits, args.slave_id)
+        modbus = ModbusRtu(
```

Comment:
Let's follow https://pypi.org/project/pep8/ for spacing.

Reply:
While I like the idea of pep8 in a general case, I could not disagree more strongly with a few of their recommendations. 

In general, I'm of the opinion that when you have a function that takes more then 2-3 arguments, using positional arguments is a really really easy way to get bugs. For calls with lots of parameters like this, Personally I find it _much_ more readable to lay them out like you would a dict or similar key-value data-type rather then just having them all in a single line.
Also, it makes things like removing or disabling one of the passed parameters _much_ easier, since you can comment out just the line with the specified parameter.

Reply:
This is valid `pep8`:

```
diff --git i/bin/modbus w/bin/modbus
index 61ecebb..ef5406f 100755
--- i/bin/modbus
+++ w/bin/modbus
@@ -39,12 +39,12 @@ class ColourHandler(logging.Handler):
 def connect_to_device(args):
     if args.device[0] == '/':
         modbus = ModbusRtu(
-                device    = args.device,
-                baud      = args.baud,
-                parity    = args.parity,
-                stop_bits = args.stop_bits,
-                slave_id  = args.slave_id,
-                timeout   = args.timeout
+                device=args.device,
+                baud=args.baud,
+                parity=args.parity,
+                stop_bits=args.stop_bits,
+                slave_id=args.slave_id,
+                timeout=args.timeout
             )
     else:
         port = 502
```
It's all about the spaces around the `=`.

The point is not liking the standard, it's avoiding wasting energy on formatting.

Reply:
I still think pep8 does things like this wrong, but hey it's not my project. pep8 version implemented in https://github.com/favalex/modbus-cli/pull/10/commits/8f72633b9e3dc500bf67ebc28c3675f1e641350e

Not having spaces around equals signs is super, _super_ terrible for readability in my personal opinion, though I won't pretend my code style preferences aren't rather idiosyncratic.

Do you have a pep8rc somewhere for the project? There's a bunch of lines-too-long warnings as well.